src/node-lib/curriculum-api-2023/fixtures/lessonMediaClips.fixture.ts

```diff
@@ -0,0 +1,217 @@
+        {
+          order: 1,
+          customTitle: "Cycle 1 Video",
+          mediaId: 137141,
+          mediaObject: {
+            url: "https://example.com/video/137141.mp4",
+            resourceType: "video",
+            displayName: "Cycle 1 Video",
+            usageRestrictions: "No restrictions",
+            attributionRequired: "No attribution required",
+          },
+          mediaType: "video",
+          videoId: 4,
+          videoObject: {
+            muxPlaybackId: "KHUK004YVrDuBlZH1YK01J2KQw2AyxGwpy00l016qIO3jDU",
+            url: "https://example.com/video/4.mp4",
+            resourceType: "video",
+            displayName: "Cycle 1 Video",
+            usageRestrictions: "No restrictions",
+            attributionRequired: "No attribution required",
+            duration: 150,
+          },
+        },
+        {
+          order: 2,
+          customTitle: "Cycle 1 Audio",
+          mediaId: 137139,
+          mediaObject: {
+            url: "https://example.com/audio/137139.mp3",
+            resourceType: "audio",
+            displayName: "Cycle 1 Audio",
+            usageRestrictions: "No restrictions",
+            attributionRequired: "No attribution required",
+          },
+          mediaType: "audio",
+          videoId: 23,
+          videoObject: {
+            muxPlaybackId: "YEPHo9rLrwNEIh02YUkNnqX6CFELAVtmlw5R4lYJESDs",
+            url: "https://example.com/video/23.mp4",
+            resourceType: "video",
+            displayName: "Cycle 1 Video",
+            usageRestrictions: "No restrictions",
+            attributionRequired: "No attribution required",
+            duration: 120,
+          },
+        },
+      ],
+      cycle2: [
+        {
+          order: 1,
+          customTitle: "Cycle 2 Audio",
+          mediaId: 137120,
+          mediaObject: {
+            url: "https://example.com/audio/137120.mp3",
+            resourceType: "audio",
+            displayName: "Cycle 2 Audio",
+            usageRestrictions: "No restrictions",
+            attributionRequired: "No attribution required",
+          },
+          mediaType: "audio",
+          videoId: 50,
+          videoObject: {
+            muxPlaybackId: "YEPHo9rLrwNEIh02YUkNnqX6CFELAVtmlw5R4lYJESDs",
+            url: "https://example.com/video/50.mp4",
+            resourceType: "video",
+            displayName: "Cycle 2 Video",
+            usageRestrictions: "No restrictions",
+            attributionRequired: "No attribution required",
+            duration: 180,
+          },
+        },
+      ],
+      cycle3: [
+        {
+          order: 3,
+          customTitle: "Cycle 3 Audio - Track 29",
+          mediaId: 137108,
+          mediaObject: {
+            url: "https://example.com/audio/137108.mp3",
+            resourceType: "audio",
+            displayName: "Cycle 3 Audio - Track 29",
+            usageRestrictions: "No restrictions",
+            attributionRequired: "No attribution required",
```

Comment:
I know we are just guessing this but I think it will always be either video or media?